src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -76,7 +77,7 @@ protected void onEvent(GHEvent event, String payload) {
         return new Function<GHEventsSubscriber, Set<GHEvent>>() {
             @Override
             public Set<GHEvent> apply(GHEventsSubscriber subscriber) {
-                return defaultIfNull(subscriber.events(), Collections.<GHEvent>emptySet());
+                return defaultIfNull(notNull(subscriber).events(), Collections.<GHEvent>emptySet());
```

Comment:
reading through this I can't help but think it would be cleaner to have some intermediate classes
NullSafePredicate, NullSafeFunction etc that perform a check then call a different method `public void nsApply(@NotNull Object)`